cargo-apk/src/apk.rs

```diff
@@ -225,13 +225,41 @@ impl<'a> ApkBuilder<'a> {
 
         let signing_key = self.manifest.signing.get(profile_name);
 
-        let signing_key = match (signing_key, is_debug_profile) {
-            (Some(signing), _) => Key {
+        let signing_key = if let Some(signing) = signing_key {
+            Key {
                 path: crate_path.join(&signing.path),
                 password: signing.keystore_password.clone(),
-            },
-            (None, true) => self.ndk.debug_key()?,
-            (None, false) => return Err(Error::MissingReleaseKey(profile_name.to_owned())),
+            }
+        } else {
+            let env_profile_name = profile_name.to_uppercase().replace('-', "_");
+
+            let path = {
+                let profile_env = format!("CARGO_APK_{}_KEYSTORE", env_profile_name);
+                std::env::var_os(&profile_env).map(PathBuf::from)
+            };
+
+            let password = {
+                let profile_env = format!(
+                    "CARGO_APK_{}_KEYSTORE_PASSWORD",
+                    profile_name.to_uppercase().replace('-', "_")
+                );
+
+                std::env::var(&profile_env).ok()
+            };
+
+            if is_debug_profile {
+                if let (Some(path), Some(password)) = (path, password) {
```

Comment:
Thinking about this some, both the path _and_ password are required. Do we silently ignore the `_KEYSTORE` env var when `_KEYSTORE_PASSWORD` isn't set, or should we `eprintln` a `warn` in that case? I can imagine users forgetting about the ""default"" `"android"` password for their debug keystore... :smile:

Reply:
Agreed, think it's reasonable for the debug store to fallback to the default password.